cpp/benchmarks/io/text/multibyte_split.cpp

```diff
@@ -103,22 +103,23 @@ static void bench_multibyte_split(nvbench::state& state)
   auto const delim_factor = static_cast<double>(delim_percent) / 100;
   auto device_input       = create_random_input(file_size_approx, delim_factor, 0.05, delim);
   auto host_input         = thrust::host_vector<char>(device_input.size());
-  auto const host_string  = std::string(host_input.data(), host_input.size());
 
   cudaMemcpyAsync(host_input.data(),
                   device_input.data(),
                   device_input.size() * sizeof(char),
                   cudaMemcpyDeviceToHost,
                   cudf::default_stream_value);
 
+  cudaDeviceSynchronize();
```

Comment:
Perhaps a better option
```suggestion
  cudf::default_stream_value.synchronize();
```